### ISSUE TITLE
upgrade to Realm 0.89.0

### ISSUE THREAD

**Issue by alexodus** [thorbenprimke/realm-recyclerview#41]
Please upgrade to Realm 0.89.0.
In my project I use this versione and now I had this error:

```
FATAL EXCEPTION: 
    java.lang.NoSuchMethodError: No virtual method get(I)Lio/realm/RealmObject; in class Lio/realm/RealmResults; or its super classes (declaration of 'io.realm.RealmResults' appears in /data/data/<package>/files/instant-run/dex/slice-io.realm-realm-android-library-0.89.0_1e0e323ab8ec35a6ebf8c8bfff373693aed24725-classes.dex)
    at io.realm.RealmBasedRecyclerViewAdapter.getRealmRowId(RealmBasedRecyclerViewAdapter.java:376)
    at io.realm.RealmBasedRecyclerViewAdapter.getIdsOfRealmResults(RealmBasedRecyclerViewAdapter.java:367)
    at io.realm.RealmBasedRecyclerViewAdapter.updateRealmResults(RealmBasedRecyclerViewAdapter.java:335)
    at io.realm.RealmBasedRecyclerViewAdapter.<init>(RealmBasedRecyclerViewAdapter.java:198)
    at io.realm.RealmBasedRecyclerViewAdapter.<init>(RealmBasedRecyclerViewAdapter.java:105)
```

**Comment by thermatk** [comment]
0.89.1 is out already
same error happens to my app if I don't downgrade to 0.88.2

**Comment by thorbenprimke** [comment]
Thanks for bringing this up. I'll see about updating the dependency.

**Comment by thorbenprimke** [comment]
closed by 67a455747b309236712ccac16c52a372ebf0fd17
